src/app/core/auth.service.ts

```diff
@@ -319,12 +320,17 @@ export class AuthService {
     if (!this.user['accessGroups']) {
       return;
     }
-    if (this.user['accessGroups'].includes('admin')) {
+
+    const groups = this.user['accessGroups'].map(x => x.split('@')[0]);
```

Comment:
If this is the solution we want to go with, can we add a comment here to explain this as it was explained in the PR description?